featuretools/tests/primitive_tests/test_upper_case_word_count.py

```diff
@@ -34,7 +37,10 @@ def test_nan(self):
             dtype="string",
         )
         primitive_func = self.primitive().get_function()
-        answers = pd.Series([pd.NA, 0.0, 2.0])
+        answers = pd.Series([pd.NA, 0, 2], dtype='Int64')
         pd.testing.assert_series_equal(
-            primitive_func(x), answers, check_names=False, check_dtype=False
+            primitive_func(x),
+            answers,
+            check_names=False,
+            check_dtype=False,
```

Comment:
```suggestion
            check_dtype=True,
```

Reply:
removed `check_dtype=False` as it defaults to True